docs/repos/git/merging-with-squash.md

```diff
@@ -24,7 +24,7 @@ The default branch is an accurate representation of the history of each topic br
 
 ## Squash merge
 
-Squash merging is a merge option that allows you to condense the Git history of topic branches when you complete a pull request. Instead of each commit on the topic branch being added to the history of the default branch, a squash merge adds all the file changes to a single new commit on the default branch. 
+Squash merging is a merge option that allows you to condense the Git history of topic branches when you complete a pull request. Instead of each commit on the topic branch being added to the history of the default branch, a squash merge adds all the file changes to a single new commit on the default branch. Squash merge commit doesn't have reference to topic branch, it is absolutely new commit containing all the changes. Furthermore it is recommended to delete topic branch to prevent confusion with it.

```

Comment:
I'd suggest to say
"reference to the topic" instead of "reference to topic"
"it is a new commit, that contains all changes from the topic branch" instead of "it is absolutely new commit containing all the changes"
"to delete the topic branch to prevent any confusion" instead of "to delete topic branch to prevent confusion with it"